astropy/utils/misc.py

```diff
@@ -503,14 +506,18 @@ class InheritDocstrings(type):
 
     For example::
 
+        >>> import warnings
         >>> from astropy.utils.misc import InheritDocstrings
-        >>> class A(metaclass=InheritDocstrings):
-        ...     def wiggle(self):
-        ...         "Wiggle the thingamajig"
-        ...         pass
-        >>> class B(A):
-        ...     def wiggle(self):
-        ...         pass
+        >>> with warnings.catch_warnings():
```

Comment:
If you're up for it, I'd keep the docstring as is, and rather skip doctests. But obviously no big deal

Reply:
I still prefer to have the doctest run because I don't want it to break before we have the chance to completely remove it. 😄